includes/class-custom-post-types.php

```diff
@@ -313,8 +331,8 @@ public function print_contact_metabox( $post ) {
 				</label>
 				<select name="contact_post" id="bpfwp_contact-page">
 					<option></option>
-					<?php foreach( $pages as $id => $title ) : ?>
-						<option value="<?php echo absint( $id ); ?>"<?php if ( $id == get_post_meta( $post->ID, 'contact_post', true ) ) : ?> selected<?php endif; ?>>
+					<?php foreach ( $pages as $id => $title ) : ?>
+						<option value="<?php echo absint( $id ); ?>"<?php if ( get_post_meta( $post->ID, 'contact_post', true ) === $id ) : ?> selected<?php endif; ?>>
```

Comment:
This created a small bug because `get_post_meta()` returns a `string` instead of an `int`. I've got it fixed here and I'll push it onto the merge.